.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "files.autoSave": "onFocusChange",
+    "conventional-branch.type": [
+      "build", // Changes that affect the build system or external dependencies
+      "ci", // Changes to our CI configuration files and scripts
+      "docs", // Documentation only changes
+      "feat", // A new feature
+      "fix", // A bug fix
+      "refactor", // A code change that neither fixes a bug nor adds a feature
+      "style", // Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+      "test", // Adding missing tests or correcting existing tests
+      "chore", // Misc things, like renaming or deleting files
```

Comment:
We could even experiment with using different ones, or a smaller subset. Like test or feat don't make that much sense here...

Reply:
That's very true!

Reply:
We could also add more for docs related things?